### ISSUE TITLE
there is no directory called SegmentationClassAug,how to solve it 

### ISSUE THREAD

**Issue by zibuyu2018** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#38]
-----------build encoder: deeplab pre-trained-----------
after start block: (10, 81, 81, 64)
after block1: (10, 81, 81, 256)
after block2: (10, 41, 41, 512)
after block3: (10, 41, 41, 1024)
after block4: (10, 41, 41, 2048)
-----------build decoder-----------
after aspp block: (10, 41, 41, 21)
INFO:tensorflow:Restoring parameters from ../reference model/deeplab_resnet.ckpt
Restored model parameters from ../reference model/deeplab_resnet.ckpt
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.NotFoundError'>, /home/hp/VOCdevkit/VOC2012/SegmentationClassAug/2008_008300.png; No such file or directory
         [[Node: create_inputs/ReadFile_1 = ReadFile[_device="/job:localhost/replica:0/task:0/device:CPU:0"](create_inputs/input_producer/Gather_1)]]

**Comment by zibuyu2018** [comment]
i have load this https://www.dropbox.com/s/oeu149j8qtbs1x0/SegmentationClassAug.zip?dl=0   and put this in VOCdevkit/VOC2012/SegmentationClassAug/.png. but there  have inconsitency label and still have this problom


thank you，
It doesn't seem to be complete, but I solved the problem by deleting some images in the txt file that I couldn't find in the SegmentationClassAug file.

**Comment by zhengyang-wang** [comment]
The augmented PASCAL VOC 2012 dataset with 10582 images for training and 1449 images for validation. Here is a download link of augmented labels: https://www.dropbox.com/s/oeu149j8qtbs1x0/SegmentationClassAug.zip?dl=0

#7

**Comment by heyitsguay** [comment]
Hi @zhengyang-wang! I have followed the instructions you provided in your most recent comment, but I still get errors about files not being found. The first is as follows, with a little initial context so you can see when it happens:

```
('after start block:', TensorShape([Dimension(10), Dimension(81), Dimension(81), Dimension(64)]))
('after block1:', TensorShape([Dimension(10), Dimension(81), Dimension(81), Dimension(256)]))
('after block2:', TensorShape([Dimension(10), Dimension(41), Dimension(41), Dimension(512)]))
('after block3:', TensorShape([Dimension(10), Dimension(41), Dimension(41), Dimension(1024)]))
('after block4:', TensorShape([Dimension(10), Dimension(41), Dimension(41), Dimension(2048)]))
-----------build decoder-----------
('after aspp block:', TensorShape([Dimension(10), Dimension(41), Dimension(41), Dimension(21)]))
Restored model parameters from ./deeplab_resnet_init.ckpt
###### This part right here
2018-06-20 00:01:55.388202: W tensorflow/core/framework/op_kernel.cc:1273] OP_REQUIRES failed at whole_file_read_ops.cc:114 : Not found: /home/matt/VOC2012/SegmentationClassAug/2008_003519.png; No such file or directory
```

I went and looked in `SegmentationClassAug`, and found that the files had names beginning with `2007_`,  and neither `2008_003519.png` nor `2007_003519.png` exists. Can you help me reconcile this discrepancy?

Thanks,
Matt

**Comment by StewartSethA** [comment]
My solution was just to delete the offending lines from the dataset/train.txt file.
[train.txt](https://github.com/zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow/files/2497648/train.txt)

**Comment by zhengyang-wang** [comment]
I have 12031 files in SegmentationClassAug. Please check.
